app/components/search.tsx

```diff
@@ -106,6 +108,17 @@ export default function Search({
                 }}
               />
             ))}
+          <a
+            href={`https://stampy.ai/wiki/Special:FormStart?form=Q&page_name=${searchInput}`}
+            target='_blank' rel="noreferrer"
+            className={`transparent-button result-item`}
+            style={{display: 'flex', alignItems: 'center', textDecoration: 'none'}}
```

Comment:
I would prefer adding a className and corresponding style in the `css` file instead of non-dynamic inline styles